controller/config.go

```diff
@@ -4,15 +4,17 @@ import (
 	"fmt"
 	"io/ioutil"
 
+	"github.com/golang/glog"
 	"github.com/pkg/errors"
 	"gopkg.in/v2/yaml"
 )
 
 func LoadFile(path string) (*Config, error) {
 	content, err := ioutil.ReadFile(path)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to read config file")
+		glog.Infof("No %s file found.  Will try to figure out defaults", path)
 	}
+
 	c, err := Load(string(content))
```

Comment:
ah looks like i had already handled the default config & empty file content... go me. looks fine then.